### PR TITLE
guake: init at 0.8.3

### PR DESCRIPTION
**Pull request by msteen**
After starting Guake it will give the error message "Guake can not init! Gconf Error. Have you installed guake.schemas properly?", which will have to be resolved manually, because I have not found a way to automate this, without being impure.

If you have Guake installed, you can use `nix-build -A gnome3.guake` to get the path to the build directory in the nix store, which then can be used in the following command to install the schemas file of Guake:
gconftool-2 --install-schema-file /path/returned/by/nix-build/share/gconf/schemas/guake.schemas

It can be removed again by the following command:
gconftool-2 --recursive-unset /apps/guake

### CODE REVIEW

**Comment by domenkozar** [comment]
cc @lethalman

**Comment by lethalman** [comment]
I really have no idea what to do about gconf. However I will take a look...

File: pkgs/development/libraries/libutempter/default.nix [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  installFlags = [
+    "libdir=\${out}/lib"
+    "libexecdir=\${out}/lib"
+    "includedir=\${out}/include"
+    "mandir=\${out}/share/man"
+  ];
+
+  meta = {
+    description = "Interface for terminal emulators such as screen and xterm to record user sessions to utmp and wtmp files";
+    license = licenses.lgpl;
```
Comment (vcunat) [inline pkgs/development/libraries/libutempter/default.nix]: This license reference is wrong, as noted on IRC now.

**Comment by DamienCassou** [comment]
@lethalman is there a reason to put this package below `gnome-3/`?

**Comment by lethalman** [comment]
It shouldn't be under gnome3 package set. It uses gconf and gtk2. It works on gnome3, but definitely not a piece resembling gnome3 or bound to any of the gnome3 packages at all.

**Comment by vcunat** [comment]
The libutemper fails to download its source.

**Comment by msteen** [comment]
@lethalman It is now under pkgs/applications/misc, just like terminator.
@vcunat Since the original source is down, I have used the original source archive used by Ubuntu.

**Comment by vcunat** [comment]
So, just the gconf error is remaining, right? (It prevents start for me.)

**Comment by vcunat** [comment]
OK, I somehow missed the work-around. Pushed with that comment, but I assume you'll (soon) improve it not to require gconf, as mentioned on IRC.

**Pull request closed without merging**